safedelete/tests/test_soft_delete_cascade.py

```diff
@@ -53,8 +56,12 @@ def setUp(self):
             Article.objects.create(author=self.authors[1]),
             Article.objects.create(author=self.authors[1], category=self.categories[1]),
             Article.objects.create(author=self.authors[2], category=self.categories[2]),
+            Article.objects.create(author=self.authors[2], category=self.categories[2])
         )
 
+        with patch.object(timezone, 'now', return_value=datetime.datetime(2012, 12, 21)) as mock_now:
```

Comment:
Remove the `mock_now` as it's unused. See the travis build log, which tests for pyflakes.

Reply:
I'd also separate this patch into a test to test for the behaviour you changed. Reason being is that you'd have to keep this in mind for all of the tests. Best to test for this separately each time imo.